tests/deployment/test_client_server.py

```diff
@@ -99,6 +99,7 @@ def test_client_server_sklearn(
     with pytest.raises(AttributeError, match=".* model is not compiled.*"):
         client_server_simulation(x_train, x_test, model, default_configuration)
 
+    # With n_bits = 3, KNN is not compilable
     fhe_circuit = model.compile(
```

Comment:
We discussed adding a test here for KNeighborsClassifier that checks that `predict` in the clear is the same as `post_processing` of the results of the server.

Reply:
I think the issue is broader than KNN right ? We want to add test for all regressors (predict vs server result) and classifiers (predict_proba vs server result, except for KNN which does not have predict_proba yet) ! 
-> https://github.com/zama-ai/concrete-ml-internal/issues/3996